xml/config/checkstyle-suppressions.xml

```diff
@@ -5,6 +5,6 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
-  <suppress files="src[\\/]generated[\\/]java[\\/]" checks=".*"/>
+  <suppress files="build[\\/]xjc[\\/]generated-sources[\\/]" checks=".*"/>
```

Comment:
I was wondering why we were using square braces everywhere. We actually do not need them.

Reply:
No idea, removing all of them from that file breaks checkstyle, so leaving them 😉

Reply:
Weird, worked for me. Anyway. If it works...

Reply:
It's a regular expression thats why Louis has no idea what it's doing.  It's matching one of "/" or "\", for windows or *nix.

Reply:
Oh! Of course... Too much Java, I thought the backslash was escaped and then escaping the slash...